debug/set_up_keyvault.sh

```diff
@@ -0,0 +1,136 @@
+# Check if User is authenticated with azure login
+
+if [ "$#" -eq 0 ]; then
+  echo "USAGE: ./set_up_keyvault.sh [\$RG_NAME] [\$DOCKER_USERNAME]"
+  exit 0
+fi
+
+# tr: Illegal byte sequence
+export LC_ALL=C
+export RG_NAME=$1
+export DOCKER_USER=$2
+export LOC=westus
+export KEYVAULT_NAME=kv$RG_NAME-$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1)
+export BASE64_FLAGS="-w 0"
+
+echo "\n Creating Resource Group \n"
+az group create \
+  --name $RG_NAME \
+  --location $LOC
+
+echo "\n Creating Key Vault \n"
+export KEYVAULT_ID=$(az keyvault create --name $KEYVAULT_NAME --resource-group $RG_NAME --location $LOC --query id -o tsv)
+
+echo "\n Setting up Folder for Certs"
+export CERT_PATH=debug/certs
+mkdir $CERT_PATH
+echo "\n Setting Up PEM Cert \n"
+export CERT1_NAME=pemCert
+openssl req -new -newkey rsa:2048 -days 365 -nodes -x509 -keyout "${CERT1_NAME}.pem" -out "${CERT1_NAME}.pem"
+az keyvault certificate import -n $CERT1_NAME --vault-name $KEYVAULT_NAME -f "${CERT1_NAME}.pem"
+# Add PEM Cert to Secrets.env
+awk '{print >out}; /-----END PRIVATE KEY-----/{out="debug/certs/cert1.cert.pem"}' out=/dev/null "${CERT1_NAME}.pem"
+export CERT1_VALUE=$(cat debug/certs/cert1.cert.pem | base64 -w 0)
+export CERT1_KEY_VALUE=$(openssl x509 -pubkey -noout -in "${CERT1_NAME}.pem"  | base64 -w 0)
```

Comment:
We are defining the flags here - https://github.com/Azure/secrets-store-csi-driver-provider-azure/pull/45/files#diff-ca9065251c9f6adb7aa95d8507779367R14, so maybe we should use that everywhere? I think we probably also need to check the host OS and configure the flag accordingly? It's `-b 0` for darwin

Reply:
If this is built inside a remote container, is it still relevant to have a flow for -w or -b?